repo/packages/Q/quobyte/2/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "packagingVersion": "2.0",
+  "name": "quobyte",
+  "version": "1.2.2",
+  "tags": [
+    "storage",
+    "filesystem",
+    "bigdata"
+  ],
+  "maintainer": "silvan@quobyte.com",
+  "description": "Quobyte is a software defined and fault tolerant distributed file system running on commodity hardware.",
```

Comment:
please also submit a PR against https://github.com/dcos/examples. For all the packages, we want to add a `getting started` doc so users understand what it is. you can reuse the format from any of the other examples there.

add a link to the example at the end of `description` property. 
Ex: `Check the package documentation at https://github.com/dcos/examples/tree/master/1.8/registry for details.`